test/objectid.test.js

```diff
@@ -131,11 +132,15 @@ describe('ObjectID', function() {
     it('should properly save an array of ObjectIDs', async () => {
       await Article.create({
         xid: objectIDLikeString,
-        xidArr: [objectIDLikeString],
+        xidArr: [objectIDLikeString, objectIDLikeString2],
         title: 'arrayOfObjectID',
       });
       const found = await Article.findOne({where: {title: 'arrayOfObjectID'}});
-      found.xidArr.should.be.an.Array().which.containDeep([new ds.ObjectID(objectIDLikeString)]);
+      // the type of the returned array is actually string even it's stored as ObjectIds in the db as expected
```

Comment:
> the type of the returned array is actually string

@agnes512 Isn't the returned `xidArr` in type objectId? From the test case, it's not string...

Reply:
I am confused too. I checked with `found.xidArr[0].should.be.an.instanceOf(ds.ObjectID)`, and it fails, while the `xid` test case in L129 _is_ an objectId. I couldn't find where the array ges converted though.

Reply:
Do you know what's the `typeof found.xidArr[0]` ?

Reply:
String!